docs/python/tutorial-django-push-to-registry.md

```diff
@@ -0,0 +1,62 @@
+---
+Order: 12
```

Comment:
I'm not sure about putting this one in the TOC, maybe have it linked from the Registry quickstart and not in the TOC?

Reply:
I don't know how this doc made it into my PR.  I would like to deny any changes from this doc.